proving/proving_options.go

```diff
@@ -67,6 +67,13 @@ func WithPowFlags(flags config.PowFlags) OptionFunc {
 	}
 }
 
+func LightMode() OptionFunc {
+	return func(o *option) error {
+		o.powFlags = config.RecommendedPowFlags()
+		return nil
+	}
+}
```

Comment:
Why don't we simplify here the same way as for the post service? `fast` and `light` are the only two options. We could just pass a boolean to `post-rs` that if `true` uses `fast` and otherwise `light`. Wdyt?

Reply:
_Fast_ and _Light_ are **not** the only two options. There could be other combinations, for example setting LARGE_PAGES, which we don't do but could _slightly_ improve performance on some setups (but won't work on most).

I wanted to keep the post-rs API as flexible as possible so it exposes flags as is. Sure, we could change it but not in this PR :pray:.